Demo/src/main/java/com/paypal/android/api/model/CardSetupToken.kt

```diff
@@ -0,0 +1,7 @@
+package com.paypal.android.api.model
+
+data class CardSetupToken(
```

Comment:
Same here, we may want to revisit in the future to keep code more DRY, but separate `CardSetupToken` and `PayPalSetupToken` types for now.